### ISSUE TITLE
[Feature Request] Don't mixup code review comments with discussion comments

### ISSUE THREAD

**Issue by Remy-Luciani** [bamlab/commentor#163]
Hello,

Pull-request is not only a place to make code review feedbacks. Sometimes it generates discussions, questions, and some developers also like to celebrate and share some love. 😁 

The issue is all these comments unrelated to code review are displayed on the dashboard making it hard to read.

I'm not your UX but here are my 2 cents: it might be cool to use a markdownish syntax specific to Commentor in order to tags comments (_e.g._, "You should make an early return here. {tag: clean-code}"), and filter these tagged comments in the dashboard table.

I'd be happy to discuss about it!

**Comment by MattAgn** [comment]
I guess one advantage of seeing non-tagged comments is to give you the ability to tag them after you wrote them on github. For example, I did it on some comments I wrote before using commentor in order to explore the use of tags without having to review 10 new pr

**Comment by Remy-Luciani** [comment]
Indeed. I was about to create a feature request to tag comments from commentor but I was really unsure. But it seems a better idea than I thought?

**Comment by sraikimaxime** [comment]
Hello there @Remy-Luciani @MattAgn  Thank you for your feedbacks ! We indeed plan on allowing the user to tag comments directly in commentor as it is described in issue #143. We do want to display comments that are not tagged yet in order to help code owners detect things that are wrong.
Nevertheless we could maybe allow users to remove comments that are not tagged on purpose ? What do you think ?

Just to clarify,  even if these comments are displayed in the table, they are not taken into account into the graphs which are supposed to be where the information is taken from :)
